bodhi/server/schemas.py

```diff
@@ -33,6 +38,8 @@
 
 
 class CSRFProtectedSchema(colander.MappingSchema):
+    """Define a mixin class to validate csrf tokens."""
```

Comment:
The docblock sounds less weird to me if you say "A mixin class..." rather than "Define a mixin class..".

Reply:
I've removed all the "Define"'s.